packages/embark/src/lib/modules/process_logs_api/index.js

```diff
@@ -12,12 +12,16 @@ class ProcessLogsApi {
   }
 
   registerAPICalls() {
+    const self = this;
     const apiRoute = '/embark-api/process-logs/' + this.processName;
     this.embark.registerAPICall(
       'ws',
       apiRoute,
       (ws, _req) => {
         this.events.on('process-log-' + this.processName, function (log) {
+          log.msg = self.escapeMessage(log.msg);
+          log.msg_clear = self.escapeMessage(log.msg_clear);
```

Comment:
Instead of introducing `self` , just replace ` function (log) {` with `(log) => {`